tests/suite/model/bibliography.typ

```diff
@@ -65,3 +65,16 @@ Now we have multiple bibliographies containing @glacier-melt @keshav2007read
 @Zee04
 
 #bibliography("/assets/bib/works_too.bib")
+
+--- issue-5614-bibliography-link-styling ---
+#show cite: set text(green)
+#show link: it => {
+  if type(it.dest) == str {
+    underline(it)
+  } else {
+    it
+  }
+}
```

Comment:
It seems that already works in typst 0.12.0. I just copy-pasted it to the webapp and it gives me green citation markers without underline, and underlined links in the bibliography as expected. Maybe I'm not getting it, but what was the bug actually? See also https://github.com/typst/typst/issues/5614#issuecomment-2576152520